File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/presenter/MeteoriteListView.java
package com.antonio.samir.meteoritelandingsspots.presenter;


import android.content.Context;

import com.antonio.samir.meteoritelandingsspots.util.GPSTracker;

public interface MeteoriteListView {
    Context getContext();

    void unableToFetch();

    void error(String s);

    void hideList();

    GPSTracker.GPSTrackerDelegate getGPSDelegate();

    void meteoriteLoadingStarted();

    void meteoriteLoadingStopped();
}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/presenter/MeteoriteListView.kt
package com.antonio.samir.meteoritelandingsspots.presenter


import android.content.Context

import com.antonio.samir.meteoritelandingsspots.util.GPSTracker

interface MeteoriteListView {
    val context: Context

    val gpsDelegate: GPSTracker.GPSTrackerDelegate

    fun unableToFetch()

    fun hideList()

    fun meteoriteLoadingStarted()

    fun meteoriteLoadingStopped()
}
